### PR TITLE
HDDS-9891. [snapshot] Update list of snapshot apis to include info details

### PR DESCRIPTION
**Pull request by jyotirmoy-gh**
## What changes were proposed in this pull request?
Update list of snapshot apis to include info details
```
bash-4.2$ ozone sh snapshot info --help
Usage: ozone sh snapshot info [-hV] <value> <snapshotName>
returns information about an existing snapshot
```

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-9891

## How was this patch tested?
NA

### CODE REVIEW

**Comment by jyotirmoy-gh** [comment]
@hemantk-12 @swamirishi @nilotpalnandi - Please review

File: hadoop-hdds/docs/content/feature/Snapshot.md [hunk 1]
```diff
@@ -57,6 +57,11 @@ SnapshotDiff CLI/API is asynchronous. The first time the API is invoked, OM star
 ozone sh snapshot listDiff [-ahV] [-s=<jobStatus>] <bucket>
 ```
 
+* Snapshot Info : Returns information about an existing snapshot
```
Comment (hemantk-12) [inline hadoop-hdds/docs/content/feature/Snapshot.md:60]:
```suggestion
* Snapshot Info: Returns information about an existing snapshot
```

**Review by hemantk-12 — COMMENTED** [review]
Thanks for the patch @jyotirmoy-gh.

LGTM. Left a minor comment.

File: hadoop-hdds/docs/content/feature/Snapshot.md (hunk 1 above)
Comment (jyotirmoy-gh) [reply hadoop-hdds/docs/content/feature/Snapshot.md]: Resolved

**Review by hemantk-12 — APPROVED** [review]

**Pull request merged**